### PR TITLE
chore: release

### PR DESCRIPTION
**Pull request by kyhyco**
**What changed? Why?**

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinbase/onchainkit",
-  "version": "0.21.9",
+  "version": "0.22.0",
```
Comment (Zizzamia) [inline package.json:3]: avoid doing this

File: src/version.ts [hunk 2]
```diff
@@ -1 +1 @@
-export const version = '0.21.9';
+export const version = '0.22.0';
```
Comment (Zizzamia) [inline src/version.ts:1]: this is correct

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## 0.22.0
+
+### Minor Changes
```
Comment (Zizzamia) [inline CHANGELOG.md:5]: avoid touching the CHANGELOG.

Touch only the one changeset file you want the GHA to use

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+
+- **feat**: refactor Identity components and add Address component. By @kyhyco #641 0903bf7
+
+  Breaking Changes
+
+  - `showAddress` has been removed from `Name` component
```
Comment (Zizzamia) [inline CHANGELOG.md:11]: Also `showAttestation` needs to be mention

**Review by Zizzamia — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (kyhyco) [reply CHANGELOG.md]: To future self, don't run `yarn release:version`

**Pull request merged**